src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/convolution_backprop_data.cpp

```diff
@@ -0,0 +1,458 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/convolution_backprop_data.hpp"
+#include "shared_test_classes/single_layer/convolution_backprop_data.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Deconvolution {
+
+const std::vector<fusingSpecificParams> fusingParamsSet{
+        emptyFusingSpec,
+};
+
+/* INSTANCES */
+/* ============= Deconvolution (Planar 2D) ============= */
+const auto convParams_ExplicitPadding_Planar_2D = ::testing::Combine(
+        ::testing::ValuesIn(kernels2d),
+        ::testing::ValuesIn(strides2d),
+        ::testing::ValuesIn(padBegins2d),
+        ::testing::ValuesIn(padEnds2d),
+        ::testing::ValuesIn(dilations2d),
+        ::testing::ValuesIn(numOutChannels_Planar),
+        ::testing::Values(ngraph::op::PadType::EXPLICIT),
+        ::testing::ValuesIn(emptyOutputPadding)
+);
+
+const std::vector<DeconvInputData> Planar_2D_inputs_smoke = {
+        DeconvInputData{
+                InputShape{{}, {{ 2, 12, 7, 7 }}},
+                ngraph::helpers::InputLayerType::CONSTANT,
+                {}
+        },
+        DeconvInputData{
+                InputShape{{-1, 12, -1, -1}, {{ 1, 12, 7, 7}, { 2, 12, 5, 7}, { 1, 12, 7, 7}}},
+                ngraph::helpers::InputLayerType::PARAMETER,
+                {{15, 15}, {9, 10}, {15, 15}}
+        }
+};
+
+const std::vector<DeconvInputData> Planar_2D_inputs_nightly = {
+        DeconvInputData{
+                InputShape{{-1, 12, -1, -1}, {{ 2, 12, 7, 7}, { 2, 12, 5, 7}, { 1, 12, 9, 4}}},
+                ngraph::helpers::InputLayerType::CONSTANT,
+                {}
+        },
+        DeconvInputData{
+                InputShape{{-1, 12, 7, 7}, {{ 1, 12, 7, 7}, { 2, 12, 7, 7}, { 1, 12, 7, 7}}},
+                ngraph::helpers::InputLayerType::CONSTANT,
+                {{15, 15}}
+        },
+        DeconvInputData{
+                InputShape{{{1, 10}, 12, 7, 7}, {{ 1, 12, 7, 7}, { 2, 12, 7, 7}, { 3, 12, 7, 7}}},
+                ngraph::helpers::InputLayerType::CONSTANT,
+                {{15, 15}}
+        },
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_Deconv_2D_Planar_FP32, DeconvolutionLayerCPUTest,
```

Comment:
It seems we only have separate test instances for Arm and x64
Does it mean we do not have any common instances, which are covered by both Arm and x64?

Reply:
@EgorDuplensky done

Reply:
@allnes The idea was to implement it as the common instances and use a filter function to drop unnecessary implementation names